### ISSUE TITLE
Problem with PORO object

### ISSUE THREAD

**Issue by phuongnd08** [rails-api/active_model_serializers#542]
So I have an Ohm object like this:

``` ruby
class MyModel < Ohm::Model
  include Ohm::DataTypes

  attribute :currency
  attribute :code

  def read_attribute_for_serialization(attr)
    self.send(attr)
  end
end
```

And a serializer

``` ruby
class MyModelSerializer < ActiveModel::Serializer
  attributes :id, :code, :currency
end
```

When I do

```
MyModelSerializer.new(MyModel[1]).to_json
```

I get

```
"{\"my_model\":{\"id\":1,\"code\":"abc",\"currency:\"USD\"}}"
```

But when I render it inside controller:

```
render json: MyModel.all, each_serializer: MyModelSerializer
```

All I get is:

```
[{"id":"1"},{"id":"2"},{"id":"3"},{"id":"4"}]
```

**Comment by therealjessesanford** [comment]
In case anyone finds this via google I have written a SO answer over here that explains how to use PORO hashes with AMS: http://stackoverflow.com/a/34056668/511168

**Comment by bf4** [comment]
@therealjessesanford This is fixed in 0.10. I'll reply in SO.
